ViMac-Swift/Services/TraverseGenericElementService.swift

```diff
@@ -54,7 +54,7 @@ class TraverseGenericElementService : TraverseElementService {
     
     private func getVisibleChildren(_ element: Element) throws -> [Element]? {
         try getChildren(element)?.filter({ child in
-            (childContainerElement?.frame.intersects(child.frame) ?? false) &&
+            (childContainerElement?.frame.intersects(child.frame) ?? true) &&
```

Comment:
childContainerElement is nil if the child element does not have a parent in parent chain that is considered a "container" (e.g. scroll area).